dbt_utils_integration_tests_howto/integration_tests__data__sql__data_generate_surrogate_key__csv.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/integration_tests/data/sql/data_generate_surrogate_key.csv b/integration_tests/data/sql/data_generate_surrogate_key.csv
+index 27a188f..a529106 100644
+--- a/integration_tests/data/sql/data_generate_surrogate_key.csv
++++ b/integration_tests/data/sql/data_generate_surrogate_key.csv
+@@ -1,5 +1,5 @@
+ column_1,column_2,column_3,expected_column_1_only,expected_all_columns
+-a,b,c,0cc175b9c0f1b6a831c399e269772661,7b193b3d33184464106f41ddf733783b
+-a,,c,0cc175b9c0f1b6a831c399e269772661,4f32a73dc87b7bbb7a654d8898d58c7e
+-,,c,f14cc5cdce0420f4a5a6b6d9d7b85f39,d9c538b129f1a3ad6ecfe55345c32a05
+-,,,f14cc5cdce0420f4a5a6b6d9d7b85f39,2fa5491950d66d153d23cfbcfea4e164
++a,b,c,0CC175B9C0F1B6A831C399E269772661,7B193B3D33184464106F41DDF733783B
++a,,c,0CC175B9C0F1B6A831C399E269772661,4F32A73DC87B7BBB7A654D8898D58C7E
++,,c,F14CC5CDCE0420F4A5A6B6D9D7B85F39,D9C538B129F1A3AD6ECFE55345C32A05
++,,,F14CC5CDCE0420F4A5A6B6D9D7B85F39,2FA5491950D66D153D23CFBCFEA4E164
```

Comment:
Why these surrogate keys are converted to uppercase?

Reply:
this is how our hash implementation generates the keys: in uppercase format

Reply:
Thank you Septi